.gitignore

```diff
@@ -8,3 +8,4 @@ releases/*.tar.gz
 releases/*.tar.gz.sha256sum
 _output/
 .vagrant/
+.idea/
```

Comment:
Add this to your global gitignore to prevent it from getting added in commits.  `git config --global core.excludesfile`

Reply:
@lengrongfu why you mark it resolved?

Reply:
Sorry. The cache show me that .idea is still here. My mistake